### DISCUSSION TITLE
Update list price through API

### DISCUSSION THREAD

**Discussion by jayallu** [amzn/selling-partner-api-models#3531] [General]
HI All,

What is the right way to update list price of a listing
1. feeds API
2. Listings API
We have tried to update the list price through feeds API with no luck. Any pointers will be helpful, thank you!

**Comment by vaidas-lungis** [comment]
Theoretical it should work with Feeds API, but it is sunsetting (check deprecation section https://github.com/amzn/selling-partner-api-docs/discussions/2951)

So Listing API is future proof, and if you want do bulk requests use `JSON_LISTINGS_FEED` - [Should I submit in bulk using the JSON_LISTINGS_FEED or individually with the Listings Items API?](https://developer-docs.amazon.com/sp-api/docs/building-listings-management-workflows-guide#should-i-submit-in-bulk-using-the-json_listings_feed-or-individually-with-the-listings-items-api)  

Also it is worth checking the `LISTING_OFFER_ONLY` requirement
https://developer-docs.amazon.com/sp-api/docs/listings-items-api-v2021-08-01-reference#requirements

**Reply by chocobn69** [reply]
Only  Feeds API v2020-09-04 is deprecated, Feeds API v2021-06-30 is not
